src/tools/rbd_wnbd/rbd_wnbd.cc

```diff
@@ -1296,6 +1323,13 @@ static int do_map(Config *cfg)
     goto close_ret;
   }
 
+  // TODO: consider substracting the time it took to perform the
+  // above operations from the following timeout.
```

Comment:
It's not immediately clear what the "following timeout" is.  I would suggest rephrasing or making `wait_mapped_disk` take a timeout argument.
```suggestion
  // above operations from cfg->image_map_timeout in
  // wait_mapped_disk().
```